### PR TITLE
Romeo, Romeo, Where for art thou :100: :100: :bug: :poop:

### PR DESCRIPTION
**Pull request by wrejcodes**
CHECK OUT THIS COOL LANGUAGE 🥇

### CODE REVIEW

File: helloworld.spl [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+
+                    Act I: Hamlet's insults and flattery.
+
+                    Scene I: The insulting of Romeo.
+
+[Enter Hamlet and Romeo]
+
+Hamlet:
+ You lying stupid fatherless big smelly half-witted coward!
```
Comment (Quinn-Donnelly) [inline helloworld.spl]: I think this could be more verbose

File: helloworld.spl [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+ codpiece. Speak your mind!
+
+[Exit Romeo]
+
+                    Scene II: Juliet and Ophelia's conversation.
+
+[Enter Ophelia]
+
+Juliet:
+ Thou art as good as the quotient between Romeo and the sum of a small
```
Comment (Quinn-Donnelly) [inline helloworld.spl]: THIS REALLLLLY HIT HOME

**Comment by Quinn-Donnelly** [comment]
@wrejcodes I think this is some of your finest work yet.

**Comment by wrejcodes** [comment]
😂 😂 😂 😂 😂 😂 😂 😂

**Comment by wrejcodes** [comment]
image
image
image

**Pull request merged**